popSimManu.tex

```diff
@@ -937,7 +937,9 @@ \section*{Supplemental Figures}
 \caption{\textbf{Migration rate parameters estimated under a two-population \emph{Drosophila} model}.
 Here we show inferred migration rates from \dadi and \fastsimcoal.
 Data were generated by simulating replicate \emph{Drosophila} genomes under the \cite{li2006inferring} model and using the genetic map
-inferred in \cite{comeron2012many}.}
+inferred in \cite{comeron2012many}.
+Directional migration from Europe to Africa is represented as MIG_AF_EU and
+migration from Africa to Europe is represented as MIG_EU_AF.}
```

Comment:
Hm, this isn't quite right, is it? The figure shows $m_\text{AF-EU}$ and $m_\text{EU-AF}$, not MIG. And, we should say what the other things are - the Ns and T_DIV?  It is obvious what these are but it's also very easy just to say, for those to whom it's not obvious.

Reply:
never mind, sorry!